PPAK_CVS/Sources/Scenes/Onboarding/OnboardingVC.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  OnboardingViewController.swift
+//  PPAK_CVS
+//
+//  Created by 김민지 on 2022/09/24.
+//
+
+import Foundation
+import UIKit
+
+import SnapKit
+import Then
+
+final class OnboardingViewController: BaseViewController {
+
+  // MARK: - Properties
+
+  private lazy var bottomCurveView = BottomCurveView().then {
+    $0.backgroundColor = Color.bottomCurveViewBgColor
+  }
+
+  private lazy var titleLabel = UILabel().then {
+    $0.onboardingExplainLabel(
+      text: Strings.Onboarding.explainTitle1,
+      textColor: Color.titleLabel,
+      font: Font.titleLabel
+    )
+  }
+
+  private lazy var descLabel = UILabel().then {
+    $0.onboardingExplainLabel(
+      text: Strings.Onboarding.explainDesc1,
+      textColor: Color.descLabel,
+      font: Font.descLabel
+    )
+  }
+
+  private lazy var skipButton = UIButton().then {
+    $0.withAlphaButton(title: Strings.Onboarding.skip, font: Font.smallButton)
+  }
+  private lazy var nextButton = UIButton().then {
+    $0.withAlphaButton(title: Strings.Onboarding.next, font: Font.smallButton)
+  }
+
+  private lazy var startButton = UIButton().then {
+    $0.withAlphaButton(
+      title: Strings.Onboarding.start,
+      font: Font.largeButton,
+      radius: Radius.largeButton
+    )
+    $0.isHidden = true
+  }
+
+  // MARK: - Life Cycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    // TODO: 건너뛰기나 시작하기 버튼을 클릭했을 때 Userdefaults 값을 저장하도록 변경하기
+    FTUXStorage().saveFTUXStatus()
+  }
+
+  // MARK: - Setup
+
+  override func setupLayouts() {
+    super.setupLayouts()
+
+    [self.bottomCurveView,
+     self.titleLabel,
+     self.descLabel,
+     self.skipButton,
+     self.nextButton,
+     self.startButton
+    ].forEach {
```

Comment:
왼쪽 구간에 <kbd>Enter</kbd>가 안들어가있네용
```suggestion
    [
      self.bottomCurveView,
      self.titleLabel,
      self.descLabel,
      self.skipButton,
      self.nextButton,
      self.startButton
    ].forEach {
```

Reply:
흠 개인적으로 이게 더 어색해 보이긴 합니다만 Enter를 넣어야하는 군요..!!

Reply:
사견이지만 열린 괄호와 닫힌 괄호를 일관되게 작성하는 게 좋다 판단했습니다. 그리고 열린괄호에 붙여 작성한다면 그 다음 줄은 2칸 띄어쓰기가 아닌 한 칸 띄어쓰기가 됩니다.
이것도 결국 취향차이라서, 해당 부분은 자율적으로 구성하는 게 좋겠네요. ㅎㅎ